### PR TITLE
LP-2196 Write unit tests for Command archive_past_courses_communities 

### PR DESCRIPTION
**Pull request by aliassad**
### Description

[LP-2196](https://philanthropyu.atlassian.net/browse/LP-2196)

Written unit tests Unit Tests for Django management command to archive course community on nodeBB.
which contains the following tests.

- `test_archive_command_with_past_courses`
- `test_archive_command_without_courses`
- `test_archive_command_with_no_past_courses`
- `test_archive_command_with_one_past_courses`

**Note:**
These changes also contains isort and linting related issues fixes.

### CODE REVIEW

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Review by qasimgulzar — APPROVED** [review]
LGTM

File: common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+import mock
+import pytz
+from django.core.management import call_command
+
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+
+class ArchivePastCoursesCommunities(ModuleStoreTestCase):
```
Comment (zeshanarifios) [inline common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py:16]: Please rename it to `TestArchivePastCoursesCommunities`

File: common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    """
+    Tests for `archive_past_courses_communities` command.
+    """
+
+    def setUp(self):
+        """
+        This function is responsible for creating courses for every test and mocking the function for tests.
+        :return:
```
Comment (zeshanarifios) [inline common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py:24]: We are following google style for docstrings Please update

File: common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        """
+        super(ArchivePastCoursesCommunities, self).setUp()
+
+        self.course_1 = CourseFactory.create(display_name='test course 1', run='Testing_course_1')
+        self.course_2 = CourseFactory.create(display_name='test course 2', run='Testing_course_2')
+        self.course_1.end = datetime.now(pytz.UTC) - timedelta(hours=2)
+        self.course_2.end = datetime.now(pytz.UTC) - timedelta(hours=2)
+
+        self.get_visible_courses_patcher = \
+            mock.patch('philu_commands.management.commands.archive_past_courses_communities.get_visible_courses')
```
Comment (zeshanarifios) [inline common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py:34]: Can we avoid \ by introducing a local variable

```
get_visible_courses = 'philu_commands.management.commands.archive_past_courses_communities.get_visible_courses'

self.get_visible_courses_patcher = mock.patch(get_visible_courses)
```

**Review by zeshanarifios — CHANGES_REQUESTED** [review]

File: common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py (hunk 1 above)
Comment (aliassad) [reply common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py:16]: Done

File: common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py (hunk 2 above)
Comment (aliassad) [reply common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py:24]: Fixed

File: common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py (hunk 3 above)
Comment (aliassad) [reply common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py:34]: modified

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

File: common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+        self.course_1 = CourseFactory.create(display_name='test course 1', run='Testing_course_1')
+        self.course_2 = CourseFactory.create(display_name='test course 2', run='Testing_course_2')
+        self.course_1.end = datetime.now(pytz.UTC) - timedelta(hours=2)
+        self.course_2.end = datetime.now(pytz.UTC) - timedelta(hours=2)
+
+        get_visible_courses = 'philu_commands.management.commands.archive_past_courses_communities.get_visible_courses'
+        archive_course_community = 'philu_commands.management.commands.' \
+                                   'archive_past_courses_communities.archive_course_community'
+
```
Comment (zeshanarifios) [inline common/djangoapps/philu_commands/management/commands/tests/test_archive_past_courses_communities_command.py:35]: How about doing this in following way.

```
        archive_past_courses_communities = 'philu_commands.management.commands.archive_past_courses_communities.'
        get_visible_courses = archive_past_courses_communities + 'get_visible_courses'
        archive_course_community = archive_past_courses_communities + 'archive_course_community'
```

**Review by zeshanarifios — CHANGES_REQUESTED** [review]
Just a minor thing, Rest of the PR LGTM 👍

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Review by zeshanarifios — APPROVED** [review]
LGTM 👍

**Pull request merged**